sweetalert2.d.ts

```diff
@@ -288,10 +288,11 @@ declare module 'sweetalert2' {
 
         /**
          * Whether or not SweetAlert2 should show a full screen click-to-dismiss backdrop.
+         * A string css color value can be passed to set the backdrop's color.
```

Comment:
I guess this comment is outdated after we decided to replace `backgroundColor` with `background`.